.circleci/config.yml

```diff
@@ -55,12 +55,27 @@ jobs:
       - run: *docs-install
       # Copy and paste this line for every submodule for which the tests pass
       - run: conda install openjpeg
-      - run: python -m "sunpy.data.sample"
       - run: python setup.py test -P time
-      # - run: python setup.py test -P net --online
       - run: python setup.py test -P physics
       - run: python setup.py test -P coordinates
+
+  astropy-time-online:
```

Comment:
You need to add this job to the workflows section at the end for it to get executed.

Reply:
Yeah. Forgot.